packages/opentelemetry-instrumentation/README.md

```diff
@@ -0,0 +1,172 @@
+# OpenTelemetry Instrumentation for web and node
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation
+```
+
+## Usage in Node
+
+```typescript
+import {
+  InstrumentationBase,
+  InstrumentationConfig,
+  InstrumentationNodeModuleDefinition,
+  InstrumentationNodeModuleFile,
+} from '@opentelemetry/instrumentation';
+
+import type * as module_name_to_be_pached from 'module_name_to_be_pached';
```

Comment:
Should this be `module_name_to_be_patched` instead? If so, this typo present throughout the example.

Reply:
yes, nice spot, thx